doc/source/operations/rabbitmq.rst

```diff
@@ -98,7 +98,7 @@ Stop all the OpenStack services which use RabbitMQ.
 
 .. code-block:: console
 
-  kayobe overcloud host command run --command "docker ps -a | egrep '(barbican|blazar|ceilometer|cinder|cloudkitty|designate|heat|ironic|keystone|magnum|manila|masakari|neutron|nova|octavia)' | awk '{ print \$NF }' | xargs docker stop"
+  kayobe overcloud host command run --command "systemctl -a | egrep '(barbican|blazar|ceilometer|cinder|cloudkitty|designate|heat|ironic|keystone|magnum|manila|masakari|neutron|nova|octavia)' | awk '{ print \$NF }' | xargs systemctl stop"
```

Comment:
Doesn't $NF work with docker ps output because the container name is the last field? systemctl -a output has a different ordering :thinking:

Reply:
This also works with systemctl as the description field is last, and the description is the same as the name.

```
root@controller0:/home/stack# systemctl -a | egrep '(barbican|blazar|ceilometer|cinder|cloudkitty|designate|heat|ironic|keystone|magnum|manila|masakari|neutron|nova|octavia)'
  kolla-cinder_api-container.service                                                                    loaded    inactive dead      docker kolla-cinder_api-container.service
  kolla-cinder_backup-container.service                                                                 loaded    inactive dead      docker kolla-cinder_backup-container.service
 ```
 
 ```
 root@controller0:/home/stack# systemctl -a | egrep '(barbican|blazar|ceilometer|cinder|cloudkitty|designate|heat|ironic|keystone|magnum|manila|masakari|neutron|nova|octavia)' |  awk '{ print $NF }'
kolla-cinder_api-container.service
kolla-cinder_backup-container.service
```

Reply:
Admittedly this is a lucky coincidence, should probably change to match on the name itself

Reply:
Done